apps/remote_control/test/lexical/remote_control/build/error_test.exs

```diff
@@ -1,41 +1,36 @@
 defmodule Lexical.RemoteControl.Build.ErrorTest do
   alias Lexical.Document
   alias Lexical.Plugin.V1.Diagnostic
-  alias Lexical.RemoteControl.Build.Error
-
-  use ExUnit.Case, async: true
+  alias Lexical.RemoteControl.Build
+  alias Lexical.RemoteControl.Build.CaptureServer
+  alias Lexical.RemoteControl.ModuleMappings
   require Logger
 
-  def compile(source) do
-    doc = Document.new("file:///unknown.ex", source, 0)
-
-    case Code.string_to_quoted(source) do
-      {:ok, quoted_ast} ->
-        try do
-          modules = for {m, _b} <- Code.compile_quoted(quoted_ast, "/unknown.ex"), do: m
-          {:ok, modules}
-        rescue
-          exception ->
-            {filled_exception, stack} = Exception.blame(:error, exception, __STACKTRACE__)
-            Logger.warning("Exception #{inspect(filled_exception)} stack: #{inspect(stack)}")
-            {doc, {:exception, filled_exception, stack, quoted_ast}}
-        end
+  use ExUnit.Case
+  use Patch
 
-      error ->
-        Logger.warning("error is #{inspect error}")
-        {doc, error}
+  setup do
+    if not Version.match?(System.version(), "~> 1.15") do
```

Comment:
is this necessary? You can start the capture server all the time and not use it.